src/ParseObject.js

```diff
@@ -627,7 +631,7 @@ export default class ParseObject {
 
     // Validate changes
     var validation = this.validate(newValues);
```

Comment:
Yeah, for sure. I'll update the request.

I'm also not thrilled with the API. I think a deferValidation: true on the constructor that just defers it for all sets + construction might be better.